src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.c

```diff
@@ -667,7 +682,8 @@ void func_80B85FA8(EnKaizoku* this, PlayState* play) {
     }
 }
 
-void func_80B86804(EnKaizoku* this, PlayState* play) {
+// Player has lost the fight
+void EnKaizoku_SetupLoss(EnKaizoku* this, PlayState* play) {
```

Comment:
I would probably suggest `SetupWin` since we try to keep things in perspective of the actor (another suggestion is `SetupPlayerLoss`).

Reply:
Yeah I was worried about the perspective, can put player in the function names.